anvil/src/eth/backend/time.rs

```diff
@@ -120,7 +120,8 @@ impl TimeManager {
         if next_timestamp <= last_timestamp {
             next_timestamp = last_timestamp + 1;
         }
-        let next_offset = if update_offset { Some(current - next_timestamp as i128) } else { None };
+        let next_offset =
+            if update_offset { Some((next_timestamp as i128) - current) } else { None };
```

Comment:
can actually do `update_offset.then(|| next_timestamp as i128 - current)`: https://doc.rust-lang.org/std/primitive.bool.html#method.then

Reply:
Thanks, I didn't know about this one! Ended up using `then_some` thanks to clippy.